src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5483,4 +5483,47 @@ This option should be used when specifying the certificate via -CertificateSubje
     <value>When creating a symbols package, allows to choose between the 'snupkg' and 'symbols.nupkg' format.</value>
     <comment>Do not localize snupkg and symbols.nupkg</comment>
   </data>
+  <data name="TrustedSignersCommandDescription" xml:space="preserve">
+    <value>Provides the ability to manage list of trusted signers located in NuGet.Config</value>
+  </data>
+  <data name="TrustedSignersCommandUsageExamples" xml:space="preserve">
```

Comment:
I don't get it. 
Why does this need localized? 
How will it get localized?

Reply:
Actually this will likely never be localized, cause of our lack of localization support for nuget.exe

Reply:
This does not need to be localized, but its a pattern we follow to put this in the resx file … see https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx#L208